modules/issue/template/template.go

```diff
@@ -311,6 +313,13 @@ func (f *valuedField) Label() string {
 	return ""
 }
 
+func (f *valuedField) HideLabel() bool {
+	if label, ok := f.Attributes["hide_label"].(bool); ok {
```

Comment:
Hmm, I'm not quite sure if we should use snake case.
Also, shouldn't it be `hide_title`?

Reply:
I was thinking the same with the name of the attr, but I went with hide_label to keep the name consistent throughout.

As for snakecase, the yaml marshal will make the map like this, and is consistent with other yaml parsing.